pkgs/by-name/cl/clippy-sarif/package.nix

```
@@ -0,0 +1,39 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  clippy,
+  clippy-sarif,
+  testers,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "clippy-sarif";
+  version = "0.4.2";
+
+  src = fetchFromGitHub {
+    owner = "psastras";
+    repo = "sarif-rs";
+    rev = "${pname}-v${version}";
+    hash = "sha256-EzWzDeIeSJ11CVcVyAhMjYQJcKHnieRrFkULc5eXAno=";
+  };
+
+  cargoHash = "sha256-F3NrqkqLdvMRIuozCMMqwlrrf5QrnmcEhy4TGSzPhiU=";
+  cargoBuildFlags = [
+    "--package"
+    pname
```

```suggestion
    "clippy-sarif"
```
substituting pname wherever possible is an anti pattern